charts/ckan/values.schema.json

```diff
@@ -0,0 +1,259 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "ckan configuration",
+  "properties": {
```

Comment:
The schema should be more strict, you basically need `additionalProperties: false` on all objects and you should add `required: []` where applicable.

Things like the securityContext should only be configurable if necessary, as the is rarely able to change something here, more often than not, it should just be hard-coded locked-down